HTML & CSS Project 4: Solo project; Backgrounds and Gradients

lmaldonadoch opened the pull request:
## HTML & CSS Project 4: Apple Website.

In this project I:

- Applied the recently acquired knowledge about backgrounds
- Used images as backgrounds and learned about its properties.
- Used gradients as backgrounds and learned how to build them and use them

SofiBretz commented:
### Changes required ♻️
### Hello @lmaldonadoch! 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

**- [ ] Consider turn 'On' your repository on the [stickler webpage](https://stickler-ci.com/)**

- [x] Consider adding the same proportion on the entire page as the original site.

![1 2](https://user-images.githubusercontent.com/42303392/77982726-3d8f8e80-72ca-11ea-9b84-7bdd52fa5904.png)

![1](https://user-images.githubusercontent.com/42303392/77982741-48e2ba00-72ca-11ea-8534-053febdc802e.png)

- [x] On 'footer' section, It should be wrapped by `<footer>` tags ([link](https://web.archive.org/web/20140301004610/http://www.apple.com/) to page).

- [x] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
* no repeating of the same rule

- [x] This section has links, consider go to the original page and check it
![2](https://user-images.githubusercontent.com/42303392/77983208-97dd1f00-72cb-11ea-9ca1-89850a072cf6.png)

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

lmaldonadoch commented:
Hi! Thank you for the comments!

- Stickler is now active and running.
- The proportion has been corrected:
![image](https://user-images.githubusercontent.com/59229758/78045869-ad862f00-7333-11ea-943d-14c6bda92dcf.png)
![image](https://user-images.githubusercontent.com/59229758/78045896-b5de6a00-7333-11ea-8a36-e09beb857c02.png)
- The footer section is now a <footer> tag
- Removed any duplicate rules on CSS by adding classes and grouping them.
- Changed the text for links.

Thanks!
